.pre-commit-config.yaml

```diff
@@ -50,7 +50,7 @@ repos:
         args: ["--config-file", "pyproject.toml"]
         exclude: |
           (?x)^.*(
-              octavia-cli/unit_tests/|
+              octavia-cli/unit_tests/
```

Comment:
can we merge this separately? I think we should align the config files when we fix the hook

Reply:
oops, this leaked unintentionaly.